src/opera/pge/base/schema/iso_metadata_measured_parameters_config_schema.yaml

```diff
@@ -0,0 +1,7 @@
+map(include('parameter'), key=str())
+---
+parameter:
+  description: str()
+  attribute_type: enum('geographicIdentifier', 'qualityInformation', 'instrumentInformation', 'sensorInformation', 'contentInformation', 'platformInformation', 'citation.identifier', 'descriptiveKeyword', 'processingInformation', 'processingParameter', 'commandLineArgument')
+  attribute_name: str(required=False)
```

Comment:
For the sake of making the purpose of this field a little clearer, lets call it `display_name`